From 5e31cce37c4e7f497bae0e74f62fd37eec3c82e4 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 28 Jul 2011 19:48:53 +0800
Subject: [PATCH] ENGR00153830-1 vpu: Add new VPU_IOC_REQ_VSHARE_MEM ioctl in mxc_vpu.h

Add new VPU_IOC_REQ_VSHARE_MEM ioctl to request vmalloced share memory.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc_vpu.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_vpu.h b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
index 287f125..355a9ef 100644
--- a/arch/arm/plat-mxc/include/mach/mxc_vpu.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
@@ -45,6 +45,7 @@ struct vpu_mem_desc {
 #define VPU_IOC_IRAM_SETTING	_IO(VPU_IOC_MAGIC, 6)
 #define VPU_IOC_CLKGATE_SETTING	_IO(VPU_IOC_MAGIC, 7)
 #define VPU_IOC_GET_WORK_ADDR   _IO(VPU_IOC_MAGIC, 8)
+#define VPU_IOC_REQ_VSHARE_MEM	_IO(VPU_IOC_MAGIC, 9)
 #define VPU_IOC_SYS_SW_RESET	_IO(VPU_IOC_MAGIC, 11)
 #define VPU_IOC_GET_SHARE_MEM   _IO(VPU_IOC_MAGIC, 12)
 
-- 
1.5.4.4

